packages/rspack/tests/configCases/split-chunks/max-size-casing/webpack.config.js

```diff
@@ -7,7 +7,6 @@ module.exports = {
 		newSplitChunks: true
 	},
 	optimization: {
-		chunkIds: "named",
```

Comment:
Why remove this?

Reply:
https://github.com/web-infra-dev/rspack/actions/runs/5187108757/jobs/9349855261?pr=3445